dev/com.ibm.ws.jpa_22_fat/test-applications/cdi/src/cdi/web/ELIServlet.java

```diff
@@ -312,6 +313,74 @@ public void testInjectionOccursBeforeInsertAndRemoveCallbacks() throws Exception
         svLogger.logp(Level.INFO, CLASS_NAME, TNAME, TNAME + "().exit");
     }
 
+    @Test
+    @Mode(TestMode.LITE)
+    public void testConverterCDIInjection() throws Exception {
+        final String TNAME = "testConverterCDIInjection";
+        svLogger.logp(Level.INFO, CLASS_NAME, TNAME, TNAME + "().enter");
```

Comment:
it shouldn't be necessary to log when a test start/ends.  This will be done automatically by FATServlet

Reply:
Removed the extra logging messages.